packages/flutter/lib/src/cupertino/text_selection_toolbar_buttons_builder.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'debug.dart';
+import 'desktop_text_selection_toolbar_button.dart';
+import 'localizations.dart';
+import 'text_selection_toolbar_button.dart';
+
+/// A Function that builds a context menu given a List of its children Widgets.
+///
+/// See also:
+///
+///   * [CupertinoTextSelectionToolbarButtonsBuilder], which receives an
+///     instance of this as a parameter.
+typedef ContextMenuFromChildrenBuilder = Widget Function(
+  BuildContext context,
+  List<Widget> children,
+);
+
+/// Calls [builder] with a List of Widgets generated by turning [buttonItems]
+/// into the default Cupertino buttons for the platform.
+///
+/// Does not build Material buttons. On non-Apple platforms, Cupertino-style
+/// buttons will still be used, because the Cupertino library does not access
```

Comment:
I'm not really sure what this means. Does that mean that on non-Apple platforms, if I'm not using Material, I get Cupertino?

Reply:
That's right.  I'll clarify.